Source/SourceKittenFramework/Clang+SourceKitten.swift

```diff
@@ -255,7 +256,8 @@ extension CXComment {
     }
 
     func commandName() -> String? {
-        return clang_BlockCommandComment_getCommandName(self).str()
+        return clang_InlineCommandComment_getCommandName(self).str() ??
```

Comment:
shouldn't we return 'block' instead of 'inline' to preserve the previous behavior?